src/yogeshNimbalkar/Test1_FirstNonRepeatChar.java

```diff
@@ -0,0 +1,25 @@
+package yogeshNimbalkar;
+
+public class Test1_FirstNonRepeatChar {
+
+	void getNonReatChar(String str) {
+		for(int index=0; index < str.length(); index++) {
+			boolean flag = true;
+			for(int index1=0; index1 < str.length();index1++) {
+				if(index != index1 && str.charAt(index)==str.charAt(index1)){
```

Comment:
here you can use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop